cmd/create/network/cmd.go

```diff
@@ -0,0 +1,157 @@
+package network
+
+import (
+	"context"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/network"
+	helper "github.com/openshift/rosa/pkg/network"
+	"github.com/openshift/rosa/pkg/ocm"
+	bsOpts "github.com/openshift/rosa/pkg/options/network"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+type NetworkSpec struct {
+	Service network.NetworkService
+}
+
+type networkStruct struct {
+	service network.NetworkService
+}
+
+func NewNetwork(spec NetworkSpec) networkStruct {
```

Comment:
If I've missed something in my above comment and we keep `networkStruct`, can we export it? I've run into issues with testing and visibility when returning non-exported types as part of exported functions.